src/query/service/src/spillers/serialize.rs

```diff
@@ -131,14 +123,51 @@ fn fake_data_schema(block: &DataBlock) -> DataSchema {
     DataSchema::new(fields)
 }
 
+fn bare_blocks_from_arrow_ipc(layout: &[usize], mut data: Buffer) -> Result<DataBlock> {
```

Comment:
https://github.com/databendlabs/databend/blob/b2ebbcec339992b914ab069f158404a2813c4af8/tests/sqllogictests/suites/query/order.test#L60

There is currently only one test case on this branch, and it feels like it could use some enhancement